components/rippling/rippling.app.mjs

```
@@ -1,11 +1,254 @@
+          value,
+        })) || [];
+      },
+    },
+    teamId: {
+      type: "string",
+      label: "Team ID",
+      description: "The ID of the team",
+      async options({ prevContext }) {
+        const args = prevContext?.next
+          ? {
+            url: next,
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)